rasa_core/broker.py

```diff
@@ -1,16 +1,53 @@
+import json
 import logging
+from typing import Any, Dict, Text, Optional
+
 import pika
-from typing import Text
+from rasa_core.utils import class_from_module_path, EndpointConfig
 
 logger = logging.getLogger(__name__)
 
 
+def load_event_channel_from_module_string(broker_config: EndpointConfig
```

Comment:
might make sense to move this method after `from_endpoint_config` for better readability